server/src/com/cloud/storage/snapshot/SnapshotManagerImpl.java

```diff
@@ -1181,9 +1181,18 @@ public boolean configure(String name, Map<String, Object> params) throws Configu
 
     @Override
     public boolean start() {
+        cleanBackingupSnaspshots();
         return true;
     }
 
+    private void cleanBackingupSnaspshots() {
```

Comment:
The below code is also present in AsyncJobManager, what is the reason for having it in both places?